jobs/port_forwarding/templates/bin/forward_ports.sh.erb

```diff
@@ -30,7 +30,7 @@ sysctl net.ipv4.conf.all.route_localnet=0
 	<% internal_ip   = rule['internal_ip']   || "127.0.0.1" %>
 	<% internal_port = rule['internal_port'] || raise("Expected non-empty 'internal_port' on '#{rule.inspect}' rule") %>
 
-	sudo iptables -t nat -A portforwarding-release -p tcp --dport <%= external_port %> -j DNAT --to <%= internal_ip %>:<%= internal_port %>
+	sudo iptables -t nat -A portforwarding-release -p tcp -d <%= spec.networks.send(spec.networks.methods(false).first).ip %> --dport <%= external_port %> -j DNAT --to <%= internal_ip %>:<%= internal_port %>
```

Comment:
or `spec.address`? https://bosh.io/docs/jobs.html#properties

It might make more sense to add support for `rule['external_ip']` though (maybe falling back to `spec.address`, unless people would genuinely want to listen on all IPs).